Makefile

```diff
@@ -56,4 +56,4 @@ release: git_version.c
 	done
 
 clean:
-	rm -rf ${all_objects} ${binaries} release.pyc release
```

Comment:
Nitpick: Should probably be part of another commit, since it has nothing to do with the update of embedded-common.

Reply:
True, removed entirely since it's pruned a few commits later anyway.